tst/EnergyPlus/unit/InputProcessor.unit.cc

```diff
@@ -74,6 +74,25 @@ namespace EnergyPlus {
 
 		}
 
+		TEST_F(InputProcessorFixture, MisleadingIDDWarningTest)
+		{
+			bool errors_found = false;
+			std::string const idd_contents = delimited_string({
+				"!IDD_Version,1.1.0",
+				"!IDD_BUILD 111111111",
+			});
+
+			ASSERT_FALSE(process_idd(idd_contents, errors_found));
+
+			std::string const idf_contents = delimited_string({
+				"Version,8.4;"
+			});
+
+			ASSERT_FALSE(process_idf(idf_contents, errors_found));
+
+			EXPECT_TRUE(compare_err_stream("   ** Severe  ** IP: Possible incorrect IDD File\r\n   **   ~~~   ** IDD_Version,1.1.0 not the same as expected =\"8.4\"\r\n", true));
```

Comment:
@rglazer2 This test is failing, because within process_idf the IDD_Version is getting reset to 8.4.0 adding the argument to use cached IDD doesn't help.  I'll send an alternate.